helm/aqua-app-enforcer/values.yaml

```diff
@@ -28,17 +30,15 @@ gate:
 
 image:
   repository: giantswarm/aqua-enforcer
-  tag: 4.6.20156
+  tag: 4.6.20236
   pullPolicy: IfNotPresent
 
 livenessProbe: {}
 readinessProbe: {}
 resources:
-  {}
-  # limits:
-  #   memory: 128Mi
-  # requests:
-  #   cpu: 100m
-  #   memory: 128Mi
+  limits:
+    memory: 1500Mi
+  requests:
+    memory: 1500Mi
```

Comment:
Shouldn't we set it to 512Mi as this is the recommended minimum for the enforcer to run?

Reply:
> Enforcer memory limit should be set to 'memory: 1500Mi' (the minimum requirement is 512Mi);

should and minimum requirement is not clear
 :)